contracts/zero-ex/contracts/src/features/multiplex/MultiplexFeature.sol

```diff
@@ -422,6 +425,13 @@ contract MultiplexFeature is
                     subcall.data,
                     inputTokenAmount
                 );
+            } else if (subcall.id == MultiplexSubcall.OTC) {
```

Comment:
Just to be clear, we're not actually using OTC in batch sells but multihop right? E.g.., this will only ever be in something like `MultiHop([BatchSell([OTC]), UniswapV3])`. Never in `BatchSell([OTC, UniswapV3])`.

Reply:
uh why not batch sells?

Reply:
Oh I thought we weren't doing fallback orders anymore

Reply:
oh yeah I asked @alexkroeger and we decided to add it to Multiplex